nss-day-cohort-48/Rare-API-Spicy-Sauce-Scouts
# Subscriptions - functions

#### Changes Made
1. Added file `filename` to `directoryname` directory.
1. Modified file `filename` to include `functionalityname` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout tlc-subscriber
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Create a fixture file for subscriptions.json and put this (or other) data in it.
[
    {
        "model": "rareapi.subscription",
        "pk": 1,
        "fields": {
            "follower": 1,
            "author": 2,
            "created_on": "2021-08-01"
        }
    },
    {
        "model": "rareapi.subscription",
        "pk": 2,
        "fields": {
            "follower": 2,
            "author": 1,
            "created_on": "2021-08-02"
        }
    }
]


4. do a python3 manage.py loaddata subscriptions
5. Go to postman and to a get http://localhost:8000/subscriptions. Make sure one of the subscriptions shows up (only 1 because they are from two different users)
6. In the body section, put in a new subscription. All you need is  one line, like the one below. You can choose whatever number you like, but needs to be one where you have a rare user. POST that to the same URL. Make sure a new subscription has been created.
{ 
"authorId": 1
}

7.  View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
